vault/core.go

```diff
@@ -2000,6 +2000,9 @@ func (s standardUnsealStrategy) unseal(ctx context.Context, logger log.Logger, c
 			return err
 		}
 	}
+	if err := c.handleVersionTimeStamps(ctx); err != nil {
```

Comment:
If this executes on a perf standby when the standby unseals, will it cause the standby to error on the storage write?

Reply:
I moved it here because standardUnsealStrategy doesn't apply to perf standbys.

Reply:
Do we want to load but not store the values on perf standbys?

Reply:
D'oh, good point.  Will fix.

Reply:
Well, I guess I'll fix in ent once this is merged.

Reply:
Decided against loading on perf standbys since it wouldn't be reliable: https://github.com/hashicorp/vault/pull/13081